src/scss/constants.scss

```diff
@@ -94,15 +96,24 @@ $spacing--xxl: 30px;
 
 $submit-button-diameter: 44px;
 
-$font-size--xxs: 10px;
+// Relative font sizes
+$font-size--document: 1rem;
+$font-size--parent: 1em;
+
+// Absolute font sizes
+$font-size--2s: 10px;
```

Comment:
The other names are using `xs` `xxs` `xxxs` same with large. Let's keep it consistent

Reply:
Having all names being same length is consistency as well as a an open way forward should there be need of more variables that could end up named like `$font-size--xxxxxl`, especially since the original is already in doubt:
```scss
$font-size--xl: 20px;
// Not sure this is the best naming for these
$font-size--xxl: 30px;
$font-size--xxxl: 50px;
```

Reply:
I know that they can end up extremely long but that's the way they are right now and we need to keep it consistent and the same. It's amateurish for some of the code like font sizes to be using `2s` `3s` and then the rest of the code `xxs` and `sm`

Reply:
> Every single change like this one should be discussed with the rest of the team.

define "like this" as to not flood the rest of the team with every line of code that has been changed

Reply:
For completeness sake, here is the full change: 
```scss
// Relative font sizes
$font-size--document: 1rem;
$font-size--parent: 1em;

// Absolute font sizes
$font-size--2s: 10px;
$font-size--xs: 11px;
$font-size--sm: 12px;
$font-size--md: 14px;
$font-size--lg: 16px;
$font-size--xl: 20px;
$font-size--2l: 30px;
$font-size--3l: 50px;

// @debt replace uses of these with the above
$font-size--xxs: $font-size--2s;
$font-size--xxl: $font-size--2l;
$font-size--xxxl: $font-size--3l;
```

And considering the amount of @debt comments throughout the project,
I'd guess "amateurish" wasn't that much of a blocker for positive changes in new code 
while simultaneously trying to not break the workings of the older.

Reply:
The border radius styles are using a different naming convention:
```tsx
$border-radius--xxs: 2px;
$border-radius--xs: 4px;
$border-radius--sm: 8px;
$border-radius--md: 12px;
$border-radius--lg: 16px;
$border-radius--xl: 24px;
$border-radius--max: 50%;
```

These should be using the same convention. If you want to improve the names we should have a discussion in Slack about the correct approach and everything should be refactored in another PR. Renaming only some of the global styles will be confusing.

Reply:
During working on a PR, some changes may be required that are on the border of the scope. There are 3 options:

1.  widening the scope of a PR which can be a slippery slope as a PR for one thing becomes a PR for changing everything
2.  keeping strictly to scope which can be a major roadblock to improving code in connected areas
3.  adding enough change, rationale and marking a line with @debt resulting in new issue/PR (possibly currently or eventually)

You're proposing an exclusive either #1 or #2 as the solutions, disregarding #3.
I have done #3.

The first discussion I propose is to determine if #3 is a way forward since the amount of @debt comments through the code base suggest it has been a viable option for a long while presumably to avoid the pitfalls of having too wide or too narrow scope.

Also relevant discussion https://github.com/sparkletown/sparkle/pull/1950#discussion_r686479953